config/prow/plugins.yaml

```diff
@@ -336,7 +336,8 @@ milestone_applier:
     main: v1.21
     master: v1.21
   kubernetes/kops:
-    master: v1.20
+    master: v1.21
+    release-1.19: v1.20
```

Comment:
I think this is incorrect. There are two 1.19 branches

Reply:
Thanks. That is a very good catch.